src/components/intent.ts

```diff
@@ -129,8 +118,13 @@ export class Intent {
         sessionPromise: Promise<ClientEncryptionSession|null>;
         sessionCreatedCallback: (session: ClientEncryptionSession) => Promise<void>;
         ensureClientSyncingCallback: () => Promise<void>;
+        origianlHomeserver: string
     };
-    private readyPromise?: Promise<unknown>;
+    private encryptionReadyPromise?: Promise<void>;
+
+    // A client that talks directly to the homeserver, bypassing pan.
```

Comment:
To reduce the knowledge one needs to have to understand this comment, I'd recommend not abbreviating pantalaimon.
```suggestion
    // A client that talks directly to the homeserver, bypassing pantalaimon.
```